File: C_Sharp_code/Example/WindowsFormsApp/VB_demo07/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        If TextBox1.Text >= 18 And TextBox2.Text = "國民" Then
            MessageBox.Show("你可以選舉")
        Else
            MessageBox.Show("你不可以選舉")
        End If
    End Sub
End Class

File: C_Sharp_code/Example/WindowsFormsApp/CS_demo07/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace CS_demo07
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            int age = Convert.ToInt32(TextBox1.Text);
            if (age>=18 && TextBox2.Text.Equals("國民"))
            {
                MessageBox.Show("你可以選舉");
            }else
                MessageBox.Show("你不可以選舉");
        }
    }
}
